README.md

```diff
@@ -0,0 +1,197 @@
+|:------------------------------------------:|:------------------------------------------:|
+| ![creating a bigtable instance][bigtable1] | ![creating a bigtable instance][bigtable2] |
+
+##### Creating a table
+
+We usually do this through an HBase shell inside Google Cloud Shell ([reference](https://cloud.google.com/bigtable/docs/quickstart-hbase)).  
+
+Go to your Google Cloud Dashboard (https://console.cloud.google.com/home/dashboard?project=YOUR-PROJECT-ID) and click on the little terminal symbol on the top right corner. That will boot your Google Shell. Wait a bit and then run the following commands:
+
+```bash
+$  curl -f -O https://storage.googleapis.com/cloud-bigtable/quickstart/GoogleCloudBigtable-Quickstart-0.9.5.1.zip
+$ unzip GoogleCloudBigtable-Quickstart-0.9.5.1.zip
+$ cd quickstart
+$ ./quickstart.sh
+```
+
+This will boot the HBase shell. You'll then issue a ```create``` command, followed by 2 quoted arguments, separated by an argument. These are the table name, and the column family. In our case, we used "test-table" as the table name and "cf1" as the column family:
+
+```
+hbase-shell> create "test-table", "cf1"
+```
+
+To see the data written in the table, later, you'll run the following command (again, inside the HBase shell):
+
+```
+hbase-shell> scan "test-table"
+```
+
+### 2. Sending events to Pub/Sub
+
+At this step, you've already created a Cloud Pub/Sub topic, either via the web interface or the helper script we provide in this repo. In our case, the topic is called "test-topic".
+
+To start the event generator, simply run the following command, on the repo's root:
+
+```bash
+$ inv generate_events --topic-name=test-topic
+```
+
+This will run an infinite loop, continuously publishing randomly generated events to our Pub/Sub topic. To stop it, Ctrl+C .
+
+### 3. Running our job on Dataflow
+
+You can either download a 'fat-jar' assembled by us, or build it yourself (if you have Scala and SBT installed).
+
+To build it, run ```$ sbt assembly```, on the repo root.
+
+You can then either enter SBT's REPL and run it from there, doing: 
+
+```bash
+sbt-repl> run --config /path/to/config/file.hocon
+```
+
+Or you can use the helper script, from the repo root:
+
+```bash
+$ inv run_project --config=/path/to/config/file.hocon
+```
+
+There's an example config file in [config/config.hocon.sample](https://raw.githubusercontent.com/snowplow/google-cloud-dataflow-example-project/release/0.1.0/config/config.hocon.sample) . If you've been using the same names as we did, you'll only need to perform some minimal changes (namely: the project ID; adjusting the Pub/Sub topic full name; setting Dataflow's correct staging location)
+
+### 4. Monitoring your job
+
+If you didn't get any Java stack traces in the output of the previous command, the Dataflow job has been properly submitted. Now there are 2 places we want to check to see if everything is running smoothly.
```

Comment:
I'm keen on removing the part about java stack traces since this shouldn't happen